src/server/game/Entities/Player/Player.cpp

```diff
@@ -771,12 +771,21 @@ uint32 Player::EnvironmentalDamage(EnviromentalDamage type, uint32 damage)
     // Absorb, resist some environmental damage type
     uint32 absorb = 0;
     uint32 resist = 0;
```

Comment:
this is unused probably, so the build fail

Reply:
```
WorldPacket data(SMSG_ENVIRONMENTALDAMAGELOG, (21));
    data << GetGUID();
    data << uint8(type != DAMAGE_FALL_TO_VOID ? type : DAMAGE_FALL);
    data << uint32(damage);
    data << uint32(absorb);
    data << uint32(resist);
    SendMessageToSet(&data, true);
```